### PR TITLE
Captaization Issue #3363

### LINKED ISSUE #3363: Normalize casing in UI text in English
We do not use a consistent casing throughout the UI.
For instance, in the "Export" menu:
* "Custom **t**abular **e**xporter" versus "SQL **E**xporter"

In the "SQL Exporter" dialog:
* "Output **e**mpty **row** (i.e. all cells null)" vs "Trim **C**olumn **N**ames"

### Expected Behavior

We should consistently use lowercase since that is what is used in most places so far.

### To solve this issue

Simply go through the existing translation files (in English) and normalize the phrases with incorrect casing.

### PR DESCRIPTION
**Pull request by pvcodes-zz**
Fixes #3363 

Changes proposed in this pull request:
- Capitalization is to be done in Sentence case

### CODE REVIEW

**Review by ostephens — CHANGES_REQUESTED** [review]
Thanks for this @pvcodes. Unfortunately it looks like you have applied quite a general change across these strings rather than checking if each string needs changing.

You can see I've commented on some of the changes - but every line you've changed in this document needs checking in this way to make sure you haven't removed uppercase letters that are correct.

File: main/webapp/modules/core/langs/translation-en.json [hunk 1]
```diff
@@ -41,72 +41,72 @@
...
     "core-index-create/sec-remaining": "$1 seconds remaining",
-    "core-index-create/almost-done": "almost done …",
-    "core-index-create/memory-usage": "Memory usage: $1% ($2/$3MB)",
+    "core-index-create/almost-done": "Almost done \u2026",
+    "core-index-create/memory-usage": "Memory usage: $1% ($2/$3mb)",
     "core-index-create/no-details": "No technical details.",
-    "core-index-create/question": "Create a project by importing data. What kinds of data files can I import?",
-    "core-index-create/formats": "TSV, CSV, *SV, Excel (.xls and .xlsx), JSON, XML, RDF as XML, and Google Data documents are all supported. Support for other formats can be added with OpenRefine extensions.",
+    "core-index-create/question": "Create a project by importing data. what kinds of data files can i import?",
+    "core-index-create/formats": "Tsv, csv, *sv, excel (.xls and .xlsx), json, xml, rdf as xml, and google data documents are all supported. support for other formats can be added with openrefine extensions.",
```
[the remark this answers is not in the document]
Comment (pvcodes) [reply main/webapp/modules/core/langs/translation-en.json:56]: I'll fix it, but need some time!

**Comment by pvcodes** [comment]
I've fixed more of the things. Have a look

**Comment by wetneb** [comment]
Personally I would only review this once @ostephens' comments above have been addressed.

**Comment by pvcodes** [comment]
@wetneb I have added more commits after that :)

**Comment by wetneb** [comment]
Maybe, but they don't seem to address the comments Owen brought up, do they? Otherwise the review comments would be marked as "Outdated" in GitHub I think.

**Comment by wetneb** [comment]
Closing this per inactivity.

**Pull request closed without merging**